recipe/meta.yaml

```diff
@@ -19,11 +19,10 @@ outputs:
       noarch: python
     requirements:
       host:
-        - python >=3.6
+        - python >=3.8
         - pip
       run:
-        - python >=3.6
-        - importlib-metadata
```

Comment:
We should probably announce to pkg maintainers that it is fine to remove importlib_metadata now.

Reply:
It's not always, I maintain some packages which use Python 3.10+ features of `importlib.metadata` so you have to continue to use `importlib_metadata`.

Reply:
Good to know. Thanks @chrisburr!

Reply:
Another case in point: vanilla python 3.11 cannot run the numpy test suite without newer(-than-bundled) `importlib_metadata`, see [here](https://github.com/conda-forge/numpy-feedstock/pull/283#issuecomment-1291711115)